package.json

```diff
@@ -68,6 +68,7 @@
     "babel-plugin-transform-es2015-classes": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-preset-react": "6.24.1",
+    "codecov": "^3.0.2",
```

Comment:
Could you pin the dep? 
It's not 100% needed because of the lock file, but we're already doing it everywhere else anyways :P